File: kyonggiyo-core/src/main/java/kyonggiyo/application/port/in/candidate/LoadCandidateUseCase.java
package kyonggiyo.application.port.in.candidate;

import kyonggiyo.application.port.in.auth.dto.UserInfo;
import kyonggiyo.application.port.in.candidate.dto.CandidateResponse;
import kyonggiyo.domain.candidate.Status;
import kyonggiyo.common.response.SliceResponse;

public interface LoadCandidateUseCase {

    SliceResponse<CandidateResponse> findAllByStatus(UserInfo userInfo, Status status, int page);


}

File: kyonggiyo-core/src/main/java/kyonggiyo/application/candidate/port/inbound/LoadCandidateUseCase.kt
package kyonggiyo.application.port.in.candidate;

import kyonggiyo.application.auth.domain.vo.UserInfo;
import kyonggiyo.application.port.in.candidate.dto.CandidateResponse;
import kyonggiyo.common.response.SliceResponse;
import kyonggiyo.domain.candidate.Status;

public interface LoadCandidateUseCase {

    SliceResponse<CandidateResponse> findAllByStatus(UserInfo userInfo, Status status, int page);


}
